projects/libyuv/libyuv_fuzzer.cc

```diff
@@ -0,0 +1,70 @@
+/* # Copyright 2020 Google Inc. */
+/* # */
+/* # Licensed under the Apache License, Version 2.0 (the "License"); */
+/* # you may not use this file except in compliance with the License. */
+/* # You may obtain a copy of the License at */
+/* # */
+/* #      http://www.apache.org/licenses/LICENSE-2.0 */
+/* # */
+/* # Unless required by applicable law or agreed to in writing, software */
+/* # distributed under the License is distributed on an "AS IS" BASIS, */
+/* # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. */
+/* # See the License for the specific language governing permissions and */
+/* # limitations under the License. */
+/* # */
+/* ################################################################################ */
+
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+#include "libyuv.h"
+using namespace std;
+
+unsigned int fastrand_seed;
+inline int fastrand()
+{
+    fastrand_seed = fastrand_seed * 214013u + 2531011u;
+    return static_cast<int>((fastrand_seed >> 16) & 0xffff);
+}
+
+
+extern "C" int LLVMFuzzerInitialize(const uint8_t *Data, size_t Size)
+{
+    srand(time(nullptr));
+    fastrand_seed = rand();
+    return 0;
+}
+
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+
+    if (size == 0)
+    {
+        libyuv::I420ToNV21(data, 0, data, 0, data, 0, nullptr, 0, nullptr, 0, 0, 0);
+        return 0;
+    }
+
+    int height = 0, width = 0, kPixels = 0, kHalfPixels = 0;
+    int range_size = (static_cast<int>(size) + 1) / 2;
+    do
+    {
+        height = fastrand() % range_size ;
+        width = fastrand() % range_size;
+        kPixels = width * height;
+        kHalfPixels = (width + 1) / 2 * (height + 1) / 2;
```

Comment:
kHalfPixels = (width + 1) / 2 * (height + 1) / 2; will evaluate the * before the / 2 ?
Suggest 
kHalfPixels = ((width + 1) / 2) * ((height + 1) / 2);

Reply:
ok